dags/ods/fb_page_insights/dags/fb_page_insights_2_bigquery.py

```diff
@@ -0,0 +1,33 @@
+"""
+Send PyconTW Facebook Page Insights Data to BigQuery
+"""
```

Comment:
nice 👍

Reply:
btw, @grimmer0125 if you get a chance, would you rename this file to `dag.py` plz ?

Reply:
Renaming is easy. I'm curious that is any convention to use `dag.py` or any advantage? Generally speaking, multiple files using the same file name in a code base is a little hard to maintain and search.

Reply:
yea you're right, it's just my convention. It's okay to keep this naming~